bc_obps/registration/tests/integration/test_operation_registration.py

```diff
@@ -0,0 +1,273 @@
+from datetime import timedelta
```

Comment:
Do we need the empty init file above?

Reply:
It aims to prevent future naming conflicts if we create a test with the same name.